### PR TITLE
Alertmanager V2 migration prep

### PR DESCRIPTION
**Pull request by vadimkerr**
# What this PR does

- Adds a Django management command and database fields required for the Alertmanager V2 migration
- Adds a post-migration warning alert

Screenshot 2024-01-19 at 17 41 04

Related to https://github.com/grafana/oncall-private/issues/2260

## Checklist

- [x] Unit, integration, and e2e (if applicable) tests updated
- [x] Documentation added (or `pr:no public docs` PR label added if not required)
- [x] `CHANGELOG.md` updated (or `pr:no changelog` PR label added if not required)

### CODE REVIEW

**Review by matiasb — APPROVED** [review]
LGTM

File: grafana-plugin/src/pages/integration/Integration.tsx [hunk 1]
```diff
@@ -311,6 +314,65 @@ class Integration extends React.Component<IntegrationProps, IntegrationState> {
...
+                  2. Check and adjust integration routes so that they match the new payload shape.
+                </Text>
+                <Text type="secondary">
+                  3. Check and adjust outgoing webhooks that use alerts from this integration so that they match the new
+                  payload shape.
+                </Text>
+                <Text type="secondary">
+                  Refer to{' '}
+                  <a href={docsURL} target="_blank" rel="noreferrer">
+                    the docs
```
Comment (brojd) [inline grafana-plugin/src/pages/integration/Integration.tsx:364]: shouldn't we show it as link so: `<Text type="link">the docs</Text>`?

Comment (vadimkerr) [reply grafana-plugin/src/pages/integration/Integration.tsx]: good catch! addressed in https://github.com/grafana/oncall/pull/3722/commits/a776c0d77b059e2960886adf51b37c48eea455a6

**Comment by Konstantinov-Innokentii** [comment]
LGTM, nice and clean!

**Review by Konstantinov-Innokentii — APPROVED** [review]

**Review by brojd — APPROVED** [review]

**Pull request merged**